EmployeeChatBot/web.config

```diff
@@ -7,7 +7,7 @@
       </handlers>
       <aspNetCore processPath="C:\Users\aquarant\source\repos\transcatcorp\EmployeeChatBot\EmployeeChatBot\bin\x64\Debug\netcoreapp3.1\EmployeeMonitoringScreen.exe" arguments="" stdoutLogEnabled="false" hostingModel="InProcess">
         <environmentVariables>
-          <environmentVariable name="ASPNETCORE_ENVIRONMENT" value="Development" />
+          <environmentVariable name="ASPNETCORE_ENVIRONMENT" value="Production" />
```

Comment:
Wouldn't it be more smarter to have a web.prod.config and use transforms with a publish profile?